### DISCUSSION TITLE
Possible to issue ria monitor CLI command from 6502?

### DISCUSSION THREAD

**Discussion by netzerohero** [picocomputer/community#82] [Q&A] [answered]
Is it possible to issue a RIA monitor CLI command string from the 6502 and have the RIA carry out the the CLI-command?

E.g., can one issue, say, the "LS" string from the 6502-side and have the RIA perform the listing of the directory?

I've been studying the current implementation architecture, and performing experiments w/o success.

If it were possible, it would open-up the entirety of the CLI-monitor commands to (say) EhBASIC (or C) issuing the CLI-command under 6502 program control as CLI-commands (present and future) are implemented.  (Like: LS, CD, STATUS, MKDIR, SET, etc.)

**Comment by rumbledethumps** [comment, marked as answer]
Blowing open access to all monitor features from the 6502 will never happen. The OS API will eventually get file management features. https://github.com/picocomputer/rp6502/issues/11 https://github.com/picocomputer/rp6502/issues/12

**Reply by netzerohero** [reply]
Thank you; I appreciate the response.
